lib/remote-storage.js

```diff
@@ -67,7 +67,7 @@ module.exports = class RemoteStorage {
       accessKeyId: creds.accessKeyId,
       secretAccessKey: creds.secretAccessKey,
       sessionToken: creds.sessionToken,
-      expiration: creds.expiration
+      expiration: creds.expiration ? new Date(creds.expiration) : undefined
```

Comment:
the expiration should always be defined, but I think this is a good extra safeguard, `undefined` works with the aws-sdk right?

Reply:
I had to add this because of a test that didn't set the expiration
new Date(undefined) results in a NaN actually